innopoints/config/dev.py

```diff
@@ -4,6 +4,7 @@
 from innopoints.config.common import *
 
 
+# SQLALCHEMY_ECHO = True  # Will log all the issued SQL statements
```

Comment:
Why is it commented out, if it is the dev config?

Reply:
Because it's ridiculous even in the dev config

Reply:
I just left it to know where to find it later, but after I'm done, I'll remove it entirely